saleor/static/dashboard-next/orders/components/OrderHistory/OrderHistory.tsx

```diff
@@ -26,84 +26,112 @@ export interface FormData {
+          });
+        case OrderEventsEmailsEnum.FULFILLMENT_CONFIRMATION:
           return i18n.t("Fulfillment confirmation has been sent to customer", {
             context: "order history message"
           });
-        case OrderEventsEmails.ORDER:
+        case OrderEventsEmailsEnum.ORDER_CONFIRMATION:
           return i18n.t("Order confirmation has been sent to customer", {
             context: "order history message"
           });
-        case OrderEventsEmails.PAYMENT:
+        case OrderEventsEmailsEnum.PAYMENT_CONFIRMATION:
           return i18n.t("Payment confirmation has been sent to customer", {
             context: "order history message"
           });
-        case OrderEventsEmails.SHIPPING:
+        case OrderEventsEmailsEnum.SHIPPING_CONFIRMATION:
           return i18n.t("Shipping details has been sent to customer", {
             context: "order history message"
           });
+        case OrderEventsEmailsEnum.TRACKING_UPDATED:
+          return i18n.t("Shipping tracking number has been sent to customer", {
+            context: "order history message"
+          });
       }
-    case OrderEvents.FULFILLMENT_CANCELED:
+    case OrderEventsEnum.FULFILLMENT_CANCELED:
       return i18n.t("Fulfillment has been cancelled", {
         context: "order history message"
       });
-    case OrderEvents.FULFILLMENT_FULFILLED_ITEMS:
+    case OrderEventsEnum.FULFILLMENT_FULFILLED_ITEMS:
       return i18n.t("Fulfilled {{ quantity }} items", {
         context: "order history message",
         quantity: event.quantity
       });
-    case OrderEvents.FULFILLMENT_RESTOCKED_ITEMS:
+    case OrderEventsEnum.FULFILLMENT_RESTOCKED_ITEMS:
       return i18n.t("Restocked {{ quantity }} items", {
         context: "order history message",
         quantity: event.quantity
       });
-    case OrderEvents.ORDER_FULLY_PAID:
+    case OrderEventsEnum.NOTE_ADDED:
+      return i18n.t("Note has been added to the order", {
+        context: "order history message"
+      });
+    case OrderEventsEnum.ORDER_FULLY_PAID:
       return i18n.t("Order has been fully paid", {
         context: "order history message"
       });
-    case OrderEvents.ORDER_MARKED_AS_PAID:
+    case OrderEventsEnum.ORDER_MARKED_AS_PAID:
       return i18n.t("Marked order as paid", {
         context: "order history message"
       });
-    case OrderEvents.OTHER:
+    case OrderEventsEnum.OTHER:
       return event.message;
-    case OrderEvents.OVERSOLD_ITEMS:
+    case OrderEventsEnum.OVERSOLD_ITEMS:
       return i18n.t("Oversold {{ quantity }} items", {
         context: "order history message",
         quantity: event.quantity
       });
-    case OrderEvents.PAYMENT_CAPTURED:
+    case OrderEventsEnum.PAYMENT_CAPTURED:
       return i18n.t("Payment has been captured", {
         context: "order history message"
       });
-    case OrderEvents.PAYMENT_REFUNDED:
+    case OrderEventsEnum.PAYMENT_FAILED:
+      return i18n.t("Payment has been failed", {
```

Comment:
I'm not sure if this is correct, I think we can simply say:

```suggestion
      return i18n.t("Payment failed", {
```

Reply:
Sounds good!